scss/mixins/_buttons.scss

```diff
@@ -30,30 +28,32 @@
   .open > &.dropdown-toggle {
     color: $color;
     background-color: $active-background;
-        border-color: $active-border;
+    border-color: $active-border;
```

Comment:
Properties should be ordered color, background-color, background-image, border-color